tracer/test/Datadog.Trace.ClrProfiler.Managed.Tests/HttpBypassTests.cs

```diff
@@ -27,7 +28,8 @@ public class HttpBypassTests
         [InlineData("https://www.datadoghq.com", false)]
         public void ShouldBypassUrlInAzureAppService(string url, bool shouldBypass)
         {
-            var exclusions = AzureAppServices.Metadata.DefaultHttpClientExclusions.Replace(" ", string.Empty).Split(',');
+            var defaultSettings = new ImmutableAzureAppServiceSettings();
```

Comment:
I think you can remove this line and the test will still run successfully

Reply:
Oh yes indeed, it's a relicate of an intermediate implem. Thanks. Done in 11bd7370730cb3629c42f1dc0b13322e2610e631